### PR TITLE
TAUR-1759 Move unit test into unit test directory, add assertions

### PR DESCRIPTION
**Pull request by oxtopus**

### CODE REVIEW

File: taurus/pipeline/scripts/refresh-remote-taurus-servers.sh [hunk 1]
```diff
@@ -230,7 +230,7 @@ pushd "${REPOPATH}"
        nta-wait-for-supervisord-stopped http://localhost:9001
      fi  &&
      if [ -f /var/run/nginx.pid ]; then
-       sudo /usr/sbin/nginx -p . -c conf/nginx-taurus.conf -s stop;
+       sudo /usr/sbin/nginx -p . -c conf/nginx-taurus.conf -g 'user root root;' -s stop;
```
Comment (vitaly-krugl) [inline taurus/pipeline/scripts/refresh-remote-taurus-servers.sh]: @oxtopus, for my own understanding, what changed to necessitate addition of the `-g` arg here? Also, if we're going to pass credentials, it should be done via environment variables as has been done with others in this and related scripts.

Comment (vitaly-krugl) [inline taurus/pipeline/scripts/refresh-remote-taurus-servers.sh]: If we're going to pass credentials, it should be done via environment variables as has been done with others in this and related scripts.

**Comment by vitaly-krugl** [comment]
@oxtopus, did something go wrong with a merge? The reason I ask is that this PR includes many changes (about 40 files' worth) that seemingly go way beyond moving and improving the single test script.

**Comment by oxtopus** [comment]
Ah.  Yes.  I conflated the branch.  Will fixup tomorrow.  Thanks for
catching!

**Comment by vitaly-krugl** [comment]
@oxtopus, per title of https://jira.numenta.com/browse/TAUR-1759 "xignite_stock_agent_test testForwardAfterHours doesn't test anything **and the script is in the wrong directory**", it appears that the entire test script consists of unit tests and needs to move to the unit test directory, not just the single test method.

File: taurus.metric_collectors/tests/unit/xignite_stock_agent_test.py [hunk 2]
```diff
@@ -446,13 +445,18 @@ def testForward(self, collectorsdb, _getLatestSample,
...
+    autospec=True)
+  @patch(
     "taurus.metric_collectors.xignite.xignite_stock_agent._getLatestSample",
     autospec=True)
   @patch(
     "taurus.metric_collectors.xignite.xignite_stock_agent.collectorsdb",
     autospec=True)
   def testForwardAfterHours(self, collectorsdb, _getLatestSample,
-                  urllib2, metricDataBatchWriter):
+      transmitMetricData, urllib2, metricDataBatchWriter):
```
Comment (vitaly-krugl) [inline taurus.metric_collectors/tests/unit/xignite_stock_agent_test.py]: The indentation is off. It should be either:

```
def testForwardAfterHours(self, collectorsdb, _getLatestSample,
                          transmitMetricData, urllib2, metricDataBatchWriter):
````

or 
```
def testForwardAfterHours(
  self, collectorsdb, _getLatestSample,
  transmitMetricData, urllib2, metricDataBatchWriter):
```

**Comment by vitaly-krugl** [comment]
LGTM

**Pull request merged**